ribs/emitters/_gradient_arborescence_emitter.py

```diff
@@ -58,6 +58,14 @@ class GradientArborescenceEmitter(EmitterBase):
             optimizer. See the gradient-based optimizers in
             :mod:`ribs.emitters.opt` for the arguments allowed by each
             optimizer. Note that we already pass in ``theta0`` and ``lr``.
+        es (str): The evolution strategy is a :class:`OptimizerBase` object
+            that is used to adapt the distribution from which new solution are
+            sampled from. This parameter must be the full or abbreviated
```

Comment:
Mention that the abbreviation can be found in `ribs.emitters.opt`, just like we mention for the ranker arg.